plenum/test/malicious_behaviors_node.py

```diff
@@ -1,25 +1,17 @@
 import random
 import types
+import common.error
```

Comment:
This changes is not using in this PR. But we can left them for future usage.

Reply:
I think we should remove this, as we already have similar functionality.